modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/DXPActivationKeysTable/components/TableKeyDetails/index.js

```diff
@@ -168,32 +168,32 @@ const TableKeyDetails = ({
 				<div className="row">
 					<div className="col-2">
 						<p className="bg-brand-primary-lighten-5 cp-key-details-paragraph px-3 py-2 rounded">
-							{getProductName(activationKeys)}
+							{getProductName(currentActivationKey)}
 						</p>
 					</div>
 
 					<div className="col-3">
 						<p className="bg-brand-primary-lighten-5 cp-key-details-paragraph px-3 py-2 rounded">
 							{getProductDescription(
-								activationKeys?.complimentary
+								currentActivationKey?.complimentary
 							)}
 						</p>
 					</div>
 
 					<div className="col-4">
 						<p className="bg-neutral-1 cp-key-details-paragraph d-flex px-3 py-2 rounded">
 							{hasVirtualClusterForActivationKeys
-								? activationKeys.maxClusterNodes
-								: activationKeys.hostName || '-'}
+								? currentActivationKey.maxClusterNodes
+								: currentActivationKey.hostName || '-'}
```

Comment:
Don't we need this verification here?
```suggestion
								? currentActivationKey?.maxClusterNodes
								: currentActivationKey?.hostName || '-'}
```